python/whylogs/viz/utils/profile_viz_calculations.py

```diff
@@ -104,6 +104,47 @@ class DatasetSummary(TypedDict):
     properties: Optional[OverallStats]
 
 
+def generate_profile_summary(
+    target_view: DatasetProfileView, config: Optional[SummaryConfig]
+) -> Optional[Dict[str, Any]]:
+    if config is None:
+        config = SummaryConfig()
+
+    if not target_view:
+        raise ValueError("This method has to get target Dataset Profile View")
+    overall_stats: OverallStats = add_overall_statistics(target_view)
```

Comment:
Not a holding comment as it tech looks good, but I noticed file is difficult to read, could you add in some white space for grouping? :)

Reply:
Done!